package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html

```diff
@@ -64,6 +64,19 @@
 		return v and translate('enabled') or translate('disabled')
 	end
 
+	local function formatBits(v, factor)
+		--to format a value given in kilobits use a factor of 1000
+		factor = factor or 1
+		units = {[0]='', 'k', 'M', 'G'}
```

Comment:
Please make all variables local.
```suggestion
		local units = {[0]='', 'k', 'M', 'G'}
```

Reply:
7618dcb07a3aa0c780d5cf975f4f3d26c8aa1b96